### PR TITLE
IITC Mobile: language change and start screen at first start

### PR DESCRIPTION
**Pull request by modos189**
The first time you start IITC Mobile, you will now see a Start screen containing basic information about how to use the application and request the necessary permissions.
https://github.com/AppIntro/AppIntro library is used for this purpose.

Screenshot_20200210_212922
Screenshot_20200210_212936

Removed 2 old pop-ups with help for using IITC Mobile.

You can skip the Start screen and then call it back from Settings.

Screenshot_20200210_213023

By default, IITC Mobile starts with localization of your device if IITC Mobile is translated into your language (otherwise, English localization is used).
On first slide of Start screen you can switch to English localization, you can also select any of the available translations in Settings.

ru_Screenshot_20200210_212852
Screenshot_20200210_213013

The list of available localizations is stored in arrays.xml → pref_languages_values

Switching between localizations is implemented using the https://github.com/akexorcist/Android-Localization library.

Menu items were previously tightly bound to the English localization. This was fixed, but when you upgrade to this version, selected menu items will be reset to default view.

### CODE REVIEW

File: mobile/app/src/main/res/values/arrays.xml [hunk 1]
```diff
@@ -63,4 +74,26 @@
...
+    <string-array name="pref_languages_values">
+        <item>cs</item>
+        <item>en</item>
+        <item>fr</item>
+        <item>de</item>
+        <item>ja</item>
+        <item>pt_BR</item>
+        <item>ru</item>
+        <item>es_419</item>
+    </string-array>
```
Comment (elkuku) [inline mobile/app/src/main/res/values/arrays.xml:118]: Seems that you are missing Spanish (es) :wink:

Comment (modos189) [reply mobile/app/src/main/res/values/arrays.xml:118]: Unfortunately, translation into Spanish is now only 25% complete.
This list will be updated as translations are ready

Comment (elkuku) [reply mobile/app/src/main/res/values/arrays.xml:118]: Oops my fault, I was looking at the translations for the "Button" :see_no_evil:

**Comment by johnd0e** [comment]
@modos189 https://iitc.app/build/test/IITC_Mobile-test.apk

**Comment by johnd0e** [comment]
I have some concerns about this PR, as new dependencies rise API level (well, Android 4.2.x is acceptable, so no problem), and inadequately increase apk size (perhaps that could be improved by some build params).

**Pull request merged**